### ISSUE TITLE
Bug: Not Capturing 10-K Item 1 text

### ISSUE THREAD

**Issue by hadoopjax** [dgunning/edgartools#45]
I've been running some tests on "Item 1" extraction across multiple symbols and have found a few that don't get picked up in the code blow. 

Code:
```
symbol = "PR" #EE also has this behavior if you need another test
filing = Company(symbol).get_filings(form="10-K").latest(1)

tenk = filing.obj()
output = tenk['Item 1']
```

**Comment by dgunning** [comment]
There are 2 different issues. The first one for **Permian Energy** the items are mostly accurate except for the first one which says **Items 1 and 2**. Will have to figure out to handle that one

<img width="831" alt="permian" src="https://github.com/dgunning/edgartools/assets/1523271/839c19f5-6980-4997-813d-9ae1ea60460e">

```python
filing = Filing(company='Permian Resources Corp', 
                cik=1658566, 
                form='10-K',
                filing_date='2024-02-29', 
                accession_no='0001658566-24-000018')
tenk=filing.obj()
tenk.items
```

**Comment by dgunning** [comment]
The second one for Excelerate Energy seems like a straightforward miss, so I will focus on that one first

**Comment by dgunning** [comment]
The Excelerate fix is done, and being tested

For Permian I'm not sure what to do about it using the current item detection method
<img width="318" alt="items1and2" src="https://github.com/dgunning/edgartools/assets/1523271/2ca8a77e-9992-4382-a36b-ac08efa43d94">

**Comment by dgunning** [comment]
Fixed in 2.21.1
- the issue with not detecting Items in normal Items "Item 1: Business"
Not fixed is Items 1 and 2: Business and Properties. The issue is that this has to be detected semantically and we need to think about it. Maybe we need to create a separate ticket
